code/__DEFINES/map_exporter.dm

```diff
@@ -0,0 +1,14 @@
+//Bits to save
+#define SAVE_OBJECTS (1 << 1)		//Save objects?
+#define SAVE_MOBS (1 << 2)			//Save Mobs?
+#define SAVE_TURFS (1 << 3)			//Save turfs?
+#define SAVE_AREAS (1 << 4)			//Save areas?
+#define SAVE_SPACE (1 << 5)			//Save space areas? (If not they will be saved as NOOP)
+#define SAVE_OBJECT_PROPERTIES (1 << 6)	//Save custom properties of objects (obj.on_object_saved() output)
```

Comment:
```suggestion
#define SAVE_OBJECTS (1 << 1) //! Save objects?
#define SAVE_MOBS (1 << 2)	//! Save Mobs?
#define SAVE_TURFS (1 << 3)	 //! turfs?
#define SAVE_AREAS (1 << 4) //! Save areas?
#define SAVE_SPACE (1 << 5)	 //! Save space areas? (If not they will be saved as NOOP)
#define SAVE_OBJECT_PROPERTIES (1 << 6)	//! Save custom properties of objects (obj.on_object_saved() output)
```

this is how documentation should be done in /tg/, doing `//!` like this allows it to be dmdocced.